security/s2a/testdata/ca.cert

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can you add a `README.md` to this folder that explains what commands you used to generate all the certs/keys? This way they can be reproduced.